WBooks/Screens/BookDetail/BookDetailView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  BookDetailView.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 05/08/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+protocol DetailBookDelegate: class {
+    func rentBook()
+}
+
+class BookDetailView: UITableViewCell, NibLoadable {
+    
+    @IBOutlet weak var bookTitle: UILabel!
+    @IBOutlet weak var bookAvaiability: UILabel!
+    @IBOutlet weak var bookAuthor: UILabel!
+    @IBOutlet weak var bookGenre: UILabel!
+    @IBOutlet weak var bookYear: UILabel!
+    @IBOutlet weak var wishlistButton: UIButton! {
+        didSet {
+            wishlistButton.layer.cornerRadius = 20
+            wishlistButton.layer.borderWidth = 1
+            wishlistButton.layer.borderColor = UIColor.blueBorder().cgColor
+            wishlistButton.setTitleColor(.blueBorder(), for: .normal)
+        }
+    }
+    @IBOutlet weak var rentButton: UIButton! {
+        didSet {
+            rentButton.layer.cornerRadius = 20
```

Comment:
If you are using the same `cornerRadius` for multiple buttons/views, you could add it to a global constant